### PR TITLE
Feature/basic setup

### PR DESCRIPTION
**Pull request by vzdrizhni**
In this project, the student builds a Restaurant Page. This project follows the lesson about building with javascript only with the help of webpack. Webpack is a static module bundler for modern JavaScript applications. Also, the purpose of the project is to display content based on a chosen tab.
⚠️ To inform: Eslint doesn't recognize ES6 "class fields" feature, hence no opportunity to fix this issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,132 @@
...
+[forks-shield]: https://img.shields.io/github/forks/vzdrizhni/RESTAURANT-PAGE.svg?style=flat-square
+[forks-url]: https://github.com/vzdrizhni/RESTAURANT-PAGE/network/members
+[stars-shield]: https://img.shields.io/github/stars/vzdrizhni/RESTAURANT-PAGE.svg?style=flat-square
+[stars-url]: https://github.com/vzdrizhni/RESTAURANT-PAGE/stargazers
+[issues-shield]: https://img.shields.io/github/issues/vzdrizhni/RESTAURANT-PAGE.svg?style=flat-square
+[issues-url]: https://github.com/vzdrizhni/RESTAURANT-PAGE/issues
+
+## 📝 License
+
+This project is [MIT](https://opensource.org/licenses/MIT) licensed.
```
Comment (mcrd25) [inline README.md:132]: Great README!

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+const path = require('path'); // eslint-disable-line no-unused-vars
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: `${__dirname}/dist`,
+    filename: 'bundle.js',
```
Comment (mcrd25) [inline webpack.config.js:7]: The instructions stated that file should be named `main.js` [See this screenshot from odin](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/dist_requirement.png)

**Review by mcrd25 — APPROVED** [review]
# Project Approved
Hi @vzdrizhni :wave: 

Great job!
Everything seems fine and I see no need for further review.

Next time please remember to follow instructions carefully. For instance if it is stated to name a file a certain name then do this.

***If you have any questions, now or in the future, do not hesitate to contact me on social media, email or on slack and I will be glad to chat with you.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

File: webpack.config.js (hunk 2 above)
Comment (vzdrizhni) [reply webpack.config.js:7]: Oh, sorry about that, thanks for your reply.

**Pull request merged**